src/models/template_report.rs

```diff
@@ -334,54 +334,6 @@ mod tests {
     use serde_json::json;
     use uuid::Uuid;
 
-    fn insert_test_run(conn: &PgConnection) -> RunData {
```

Comment:
It doesn't look like this is needed, but I wanted to make sure this removal was intentional.

Reply:
Yup, it was just a redundant bit of test code that I noticed was left over from some changes I made before.